tests/__init__.py

```diff
@@ -116,8 +119,24 @@ async def _create_db():
     return db
 
 
+class _ActiveDirectoryLDAPServer(LDAPServer):
+    """LDAPServer implementation with Active Directory
+    specific attribute rootDomainNamingContext"""
```

Comment:
I found this very hard to understand.

```suggestion
    """Extends LDAPServer to return AD-specific attributes
    
     Includes `rootDomainNamingContext` in bind responses.
     """
```